test/lightning_web/end_to_end_test.exs

```diff
@@ -188,4 +293,38 @@ defmodule LightningWeb.EndToEndTest do
       return state;
     });"
   end
+
+  defp start_runtime_manager(_context) do
+    rtm_args = "node ./node_modules/.bin/worker -- --backoff 0.5/5"
+
+    Application.put_env(:lightning, RuntimeManager,
+      start: true,
+      args: String.split(rtm_args),
+      cd: Path.expand("../../assets", __DIR__)
+    )
+
+    {:ok, rtm_server} = RuntimeManager.start_link(name: E2ETestRuntimeManager)
+
+    running =
+      Enum.any?(1..20, fn _i ->
+        Process.sleep(50)
+        %{runtime_port: port} = :sys.get_state(rtm_server)
```

Comment:
Should we not have a public interface for this?

Reply:
I think it's fine for now, just for the test setup.